extensions/amp-analytics/0.1/vendors/catchpoint.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const CATCHPOINT_CONFIG = /** @type {!JsonObject} */ ({
+  'requests': {
+      'endpoint': 'https://r.3gl.net/hawklogserver/r.p',
```

Comment:
Hi @calebcordry I have updated the code, could you please review and let me know if this can be pushed.
Thanks!

Reply:
Thanks for fixing the space. You are still not using `endpoint` here, only the `pageview` request. You can delete endpoint to save bytes.

Reply:
Yes, we do use it.

Reply:
Can we have this merged to the prod branch, or do we have to wait for the CLA?

Reply:
You will need to sign the CLA before we can merge. Sorry if I was not clear earlier, you are not currently using the defined `'endpoint'` in your current configuration. You are currently only using the `pageview` key in the requests object, which appears to have the same url as `endpoint` plus more.

Reply:
Thanks for the pointing out, I have updated the code. I will get the CLA signed soon.
Question on CLA, I am working with my manger to have the company sign the CLA however I have committed code using my personal account.
It would be great if you can help me out here, we want this rollout soon to our clients.
Thanks

Reply:
This can be tricky. You will have to change the commit author to be your corporate email. You can do this by rebasing but be careful. Here is one article trying to walk you through it. https://www.git-tower.com/learn/git/faq/change-author-name-email